pgstac_ingestor/collection.py

```diff
@@ -17,10 +17,11 @@ def ingest(collection: StacCollection):
         creds = get_db_credentials(os.environ["DB_SECRET_ARN"])
         with PgstacDB(dsn=creds.dsn_string, debug=True) as db:
             loader = Loader(db=db)
-            collection = [
-                collection.to_dict()
-            ]  # pypgstac wants either a string or an Iterable of dicts.
-            loader.load_collections(file=collection, insert_mode=Methods.upsert)
+            loader.load_collections(
+                # pypgstac wants either a string or an Iterable of dicts.
+                file=[collection.to_dict()],
```

Comment:
error was that we were changing the type of `collection`
in reality there is no real need to create a new variable here

Reply:
I think Mypy was complaining about the `type` of collection.